### ISSUE TITLE
Getting toast message in multiple Component

### ISSUE THREAD

**Issue by nirajhi** [magicismight/react-native-root-toast#100]
I am using react-native-navigation as my router. When I call the Toast Message in the sidebar, I get the Toast on both the sidebar and Main menu. Is there a way to stop this.

![Screen Shot 2019-03-21 at 1 56 14 PM](https://user-images.githubusercontent.com/2114194/54739956-f0baa780-4be1-11e9-9262-cc4e8ad0f8e7.png)

**Comment by olegdev** [comment]
Same thing. Any ideas?

**Comment by hlus** [comment]
I have the same issue.

**Comment by Amenocy** [comment]
same 
![image](https://user-images.githubusercontent.com/2260390/58762311-8619d680-8557-11e9-99fb-40c7ee139a51.png)

**Comment by hlus** [comment]
Yeah, I think this toast is created in each "registered component".
I use React Native Navigation (Wix) and I have custom topBar.
I call show toast and I see 2 toasts in header and screen.
Temporary solution:
set position is -70 or more and this will be shift the toast to the some space where it is will be visible only шт one place.

**Comment by FinnGu** [comment]
Also happening when displaying a modal with transparent background with react-native-navigation.
